### ISSUE TITLE
Updates that are auto-pushed to batched do not get a "push to stable" button

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1907]
Michael Cronenworth reported today that updates that were auto-pushed to batched do not get a "push to stable" button:

> On 10/06/2017 01:47 PM, Randy Barlow wrote:
> > but either way you can push it to batched,
> > and you can push it to stable too.
> 
> Except when it was auto-pushed to batched.
> 
> https://bodhi.fedoraproject.org/updates/FEDORA-2017-a62dd57720
> 
> - Auto-pushed to batched
> - No push-to-stable button
> - After editing to mark as Urgent, still no push-to-stable button
> 
> Am I forced to wait an entire week to let it be pushed out via batch?

I looked into it a bit, and the problem seems to be in [```Update.meets_testing_requirements()```](https://github.com/fedora-infra/bodhi/blob/2.12.2/bodhi/server/models.py#L2447-L2479), which the Update template uses to decide whether to render that button:

```
        # non-autokarma updates have met the testing requirements if they've reached the karma
        # threshold.
        if not self.autokarma and self.stable_karma not in (0, None)\
                and self.karma >= self.stable_karma:
            return True
```

I think we can *probably* remove that ```self.autokarma``` from the ```if``` statement to solve this, but I'm not 100% sure that won't cause other side effects. As I've written, Bodhi's state machine code is extremely messy as it's scattered all over the code base, so it can be very tricky to edit without causing unexpected side effects (in fact, this very bug is an unexpected side effect that resulted from editing the state machine code.)

**Comment by bowlofeggs** [comment]
Adam Williamson suggested a workaround of using the CLI to change the Update's state, and that does appear to work on my development box.

**Comment by bowlofeggs** [comment]
This patch is included in the bodhi-3.1.0-0.1.beta, which is deployed to [staging](https://bodhi.stg.fedoraproject.org/) and is also available via [Copr](https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release/).
